\documentclass{article}
\usepackage{fullpage}
\usepackage{hyperref}
\usepackage{authblk}
%\VignetteIndexEntry{Using aracne.networks}

\title{aracne.networks, a data package containing gene regulatory networks assembled from TCGA data by the ARACNe algorithm}
\author[1,2]{Federico M. Giorgi}
\author[1,3]{Mariano J. Alvarez}
\author[1]{Andrea Califano}
\affil[1]{Department of Systems Biology, Columbia University, New York, USA}
\affil[2]{CRUK, Cambridge University, Cambridge, UK}
\affil[3]{DarwinHealth Inc., New York, USA}
\date{\today}

\begin{document}
\SweaveOpts{concordance=TRUE}
\maketitle


<<echo=FALSE>>=
library(aracne.networks)
@


%-----------
\section{Overview of aracne.networks data package}\label{sec:overview}
The \emph{aracne.networks} data package provides context-specific transcriptional regulatory networks (also called interactomes or regulons) reverse engineered by the ARACNe algorithm from The Cancer Genome Atlas (TCGA) RNAseq expression profiles.

\paragraph{ARACNe networks}
This package contains \Sexpr{nrow(data(package='aracne.networks')$results)} Mutual Information-based networks assembled by ARACNe-AP \cite{Giorgi2016} with default parameters (MI p-value = $10^{-8}$, 100 bootstraps and  permutation seed = 1). ARACNe is a network inference algorithm based on an Adaptive Partioning (AP) Mutual Information (MI) approach \cite{Giorgi2016}. In short, ARACNe-AP estimates all pairwise Mutual Information scores between gene expression profiles, then assesses the significance of such Mutual Information by comparison to a null dataset. ARACNe then draws network edges between centroid genes (Transcription Factors and Signaling Proteins) and genes significantly associated with them (i.e. with significant MI). It then calculates Data Processing Inequality (DPI) to reduce the number of indirect connections.

ARACNe-AP was run on RNA-Seq datasets normalized using Variance-Stabilizing Transformation \cite{Anders2010}.
The raw data was downloaded on April 15\textsuperscript{th}, 2015 from the TCGA official website \cite{Weinstein2013}.
We follow the TCGA naming convention (e.g. BRCA = Breast Carcinoma) to name the individual context-specific networks.

<<echo=TRUE, results=verbatim>>=
library(aracne.networks)
data(package="aracne.networks")$results[, "Item"]
@

\paragraph{Write a network to file}
The package contains a function to print individual networks into a file.
Four columns will be printed: the Regulator id, the Target id, the Mode of Action (MoA, inferred by Spearman correlation analysis \cite{Alvarez2016}) that indicates the sign of the association between regulator and target gene and ranges betrween -1 and +1, the Likelihood (essentially an edge weight that indicates how strong the mutual information for an edge is when compared to the maximum observed MI in the network, it ranges between 0 and 1). Further details about the \emph{regulon} object as a model for transcriptional regulation are present in the manuscript \cite{Alvarez2016}.

In the following example, we print the first 10 interactions from the bladder carcinoma (blca) network. The network genes are identified by Entrez Gene ids.
<<echo=TRUE, results=verbatim>>=
data(regulonblca)
write.regulon(regulonblca, n = 10)
@
The user may want to analyze all the connections of a particular regulator (E.g. "399", the RHOH gene).
<<echo=TRUE, results=verbatim>>=
data(regulonblca)
write.regulon(regulonblca, regulator="399")
@



%-----------
\begin{thebibliography}{00}
\bibitem{Giorgi2016} Giorgi,F.M. et al. (2016) ARACNe-AP: Gene Network Reverse Engineering through Adaptive Partitioning inference of Mutual Information. Bioinformatics doi: 10.1093/bioinformatics/btw216.
\bibitem{Anders2010} Anders, S and Huber W. (2010) Differential expression analysis for sequence count data. Genome Biol 2010;11(10):R106
\bibitem{Weinstein2013} Weinstein J.N. et al. (2013) The cancer genome atlas pan-cancer analysis project. Nature Genetics 45, 1113-1120 2013
\bibitem{Alvarez2016} Alvarez M.J. et al. (2016) Functional characterization of somatic mutations in cancer using network-based inference of protein activity. Nature Genetics \emph{in press}.
\end{thebibliography}

%----------
\end{document}


